rdtho2525/Niche_GoT
# Feature/trivia round

## Is this a fix or a feature?
  + feature

## What is the change?
  + Adds trivia gameplay functionality to the app

## Where should the reviewer start?
  + src/components

## How should this be tested?
  + visit http://localhost:3000/
  + on load, clicking the "Next Quote" button should reveal a new quote and four answer buttons
  + clicking on a correct answer should reveal affirming feedback
      + should also add a green background color to the selected card
  + clicking on an incorrect answer should reveal the appropriate feedback
     + should also add a red background color to the selected card and reveal the corrected answer, colored in green
  + clicking "Next Quote" once again should reset the feedback and the answers to their original state with a new quote and four new answer values
